README.md

```diff
@@ -16,6 +16,13 @@ Download the project from Github, compile using
 ``` java 
 >>> javac MAP.java MONSTRO.java EXPLORER.java COMIDA.java GameScreenCli.java
 ```
+
+To install dependencies, maven and java 1.8 are needed, then you can run: 
+
+```sh
+mvn clean install 
+```
+
 -------
```

Comment:
Can you change to:
mvn clean compile package
java -jar target\explorer-1.0.jar